### DISCUSSION TITLE
Mastodon account for this project

### DISCUSSION THREAD

**Discussion by linux-lukas** [jellyfin/jellyfin#7811] [General]
Could the jellyfin project set up a Mastodon account for this project? I think it would help the Jellyfin project a lot in general.

**Comment by crobibero** [comment]
Curious- how would it benefit the Jellyfin project?

Note that we currently have a subreddit and a matrix based chat.

I believe we also have a Facebook and Twitter, with the Facebook group not very active.

**Reply by linux-lukas** [reply]
It would help the Jellyfin project to the extent that people from the Fediverse would become aware of this project and, in the best case, buy a Raspberry Pi to install Jellyfin on it in order to get away from Netflix, Spotify and co in the long and medium term (not only for financial reasons). In addition, there would be the potential possibility that they would tell their family and friends about the project and more and more people would find out about this super project here.

Facebook is not exactly talented in terms of compliance with European data protection law and is justifiably criticised. :/

Mastodon, which is part of Fediverse (see https://en.wikipedia.org/wiki/Fediverse), claims to be a social media where users have complete control over their data. No advertising, no tracking, open source and decentralised.
